gcloud/bigquery/test_client.py

```diff
@@ -74,27 +74,11 @@ def test_list_datasets_defaults(self):
         self.assertEqual(req['method'], 'GET')
         self.assertEqual(req['path'], '/%s' % PATH)
 
-    def test_list_datasets_explicit(self):
```

Comment:
It seems you deleted the old test when you probably just wanted to add the new one.